core/src/main/scala/org/apache/spark/rdd/SubtractedRDD.scala

```diff
@@ -48,7 +48,7 @@ import org.apache.spark.serializer.Serializer
  * you can use `rdd1`'s partitioner/partition size and not worry about running
  * out of memory because of the size of `rdd2`.
  */
-private[spark] class SubtractedRDD[K: ClassTag, V: ClassTag, W: ClassTag](
```

Comment:
These ClassTags should not be removed. I just pushed an update that reverts this line change.